client/components/seo-preview-pane/index.jsx

```diff
@@ -7,15 +7,18 @@ import React, { PropTypes } from 'react';
 import PureComponent from 'react-pure-render/component';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
-import { get } from 'lodash';
+import { compact, find, get } from 'lodash';
```

Comment:
I didn't mention this before, but when we have multiple named imports, we like to separate them onto their own lines. Maybe this isn't a big deal if it ends here, but it's the same thing for code diffs as the trailing comma is - having them on one line can make for noise. It's a subjective call here whether or it's worth it or not though.